src/WinRT.Runtime/ObjectReference.cs

```diff
@@ -333,47 +324,39 @@ internal unsafe bool IsReferenceToManagedObject
         {
             get
             {
-                return VftblIUnknown.Equals(IUnknownVftbl.AbiToProjectionVftbl);
+                ThrowIfDisposed();
```

Comment:
`ThisPtr` already checks for disposal, can we not remove this? I mean, is it not redundant?

Reply:
Yeah we can remove this since the method is private anyway.